src/app/charts/core/files/config/tpl/arkcase-portal-server.yaml

```diff
@@ -17,7 +17,7 @@ portal:
   uiApplicationHost: {{ printf "https://%s" $baseUrl.hostname | quote }}
   id: {{ $portal.portalId | quote }}
   url: *url
-  arkcaseUrl: "https://localhost:8443/arkcase"
+  arkcaseUrl: "https://core:8443/arkcase"
```

Comment:
Let's not use a hardcoded URL, and instead use `"${ARKCASE_ADMIN_URL}"` as the target string.

Also, remember to bump the chart version numbers in the relevant `Chart.yaml` files.